cmd/pcr0tool/commands/printnodes/command.go

```diff
@@ -82,6 +82,7 @@ func (cmd Command) Execute(args []string) {
 	}
 	imagePath := args[0]
 
+	fianoUEFI.DisableDecompression = false
```

Comment:
It appears `printnodes` wasn't printing compressed nodes since the `DisableCompression` optimization was merged. Fixing it.

Reply:
Could you please add this as a coment?